BRANDING.MD

```diff
@@ -1,11 +1,11 @@
 ## Knative Brand Guidelines
 
-The Knative trademark and ![Knative Logo](icons/logo.svg "The Knative Logo")
-logo ("Knative Trademarks") are trademarks of Google, and are treated
-separately from the copyright or patent license grants contained in the
-Apache-licensed Knative repositories on GitHub. Any use of the Knative
-Trademarks other than those permitted in these guidelines must be approved in
-advance. 
+The Knative trademark, ![Knative Logo](icons/logo.svg "The Knative Logo")
+logo ("Knative Trademarks") and knative.dev domain are trademarks of Google,
```

Comment:
Just a question, do we need to do anything special to get the knative.dev trademarked? I see that we currently only have knative trademarked, so just curious.